src/apis/itask_management_api.ts

```diff
@@ -0,0 +1,97 @@
+import {IIdentity} from '@essential-projects/iam_contracts';
+
+import {TaskList} from '../data_models/tasks/index';
+
+/**
+ * The ITaskManagementApi is used to retreive and manage Tasks.
+ */
+export interface ITaskManagementApi {
+
+  /**
+   * Retrieves a list of all suspended Tasks.
+   *
+   * @async
+   * @param  identity            The requesting users identity.
+   * @returns                    A list of all waiting Tasks.
+   *                             Will be empty, if none are available.
+   * @throws {UnauthorizedError} If the given identity does not contain a
+   *                             valid auth token.
+   * @throws {ForbiddenError}    If the user is not allowed to access the
+   *                             ProcessModel.
+   */
+  getAllSuspendedTasks(identity: IIdentity): Promise<TaskList>;
+
+  /**
+   * Retrieves a list of all suspended Tasks belonging to an instance of a
+   * specific ProcessModel.
+   *
+   * @async
+   * @param  identity            The requesting users identity.
+   * @param  processModelId      The ID of the ProcessModel for which to
+   *                             retrieve the Tasks.
+   * @returns                    A list of waiting Tasks for the given
+   *                             ProcessModel.
+   *                             Will be empty, if none are available.
+   * @throws {UnauthorizedError} If the given identity does not contain a
+   *                             valid auth token.
+   * @throws {ForbiddenError}    If the user is not allowed to access the
+   *                             ProcessModel.
+   */
+  getTasksForProcessModel(identity: IIdentity, processModelId: string): Promise<TaskList>;
```

Comment:
```suggestion
  getSuspendedTasksForProcessModel(identity: IIdentity, processModelId: string): Promise<TaskList>;
```

Ich weis, das ist in den anderen Klassen auch nicht so umgesetzt. Sollte man aber mal nachziehen.